.github/workflows/perfs.yml

```diff
@@ -1,24 +1,59 @@
 name: Performance tests
+
 on:
   pull_request:
-    types: [labeled]
+    types: [opened, synchronize, reopened]
+
+# Abort if new commit since then
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
 
 jobs:
   perf-tests:
-    if: ${{ github.event.label.name == 'Performance checks' }}
+    if: ${{ !contains(github.event.issue.labels.*.name, 'skip-performance-checks') }}
```

Comment:
it's github.event.pull_request here

Reply:
:O I must have copy-pasted something at some point without realizing

Reply:
thanks

Reply:
That was it!!!